### ISSUE TITLE
ppc64le tests fail with gcc-4.8.5 without flags

### ISSUE THREAD

**Issue by junaruga** [nemequ/simde#202]
When running on the latest master branch commit: `dec4b22cfdf5538eaef025b4149eb03d31788033` with gcc-4.8.5 on ppc64le EPEL7 (= CentOS7) environment, 

```
+ gcc --version
gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39)
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

$ g++ --version
g++ (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39)
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

```
+ mkdir test/build-gcc
+ pushd test/build-gcc

+ CC=gcc
+ CXX=g++
+ meson ../..
The Meson build system
Version: 0.47.2
...
+ ninja-build -v
...
+ ./test/run-tests
Running test suite with seed 0x247ffa36...
...
/arm/neon/vadd_s8/emul/c             [ ERROR ]
/arm/neon/vadd_s16/emul/c            Error: ../../test/arm/neon/vadd.c:92: assertion failed: ((const int8_t*) &(r))[8] == ((const int8_t*) &(test_vec[i].r))[8] (0 == 12)
Error: child killed by signal 6 (Aborted)
...
/arm/neon/vdup_n_s8/emul/c           [ ERROR ]
/arm/neon/vdup_n_s16/emul/c          Error: ../../test/arm/neon/vdup_n.c:67: assertion failed: ((const int8_t*) &(r))[8] == ((const int8_t*) &(test_vec[i].r))[8] (0 == -16)
Error: child killed by signal 6 (Aborted)
...
/arm/neon/vsub_s8/emul/c             [ ERROR ]
/arm/neon/vsub_s16/emul/c            Error: ../../test/arm/neon/vsub.c:92: assertion failed: ((const int8_t*) &(r))[8] == ((const int8_t*) &(test_vec[i].r))[8] (0 == -4)
Error: child killed by signal 6 (Aborted)
...
956 of 990 (97%) tests successful, 0 (0%) test skipped.
```

All the log is [here](https://gist.github.com/junaruga/d92f419219ac0494142f3dccaf3a9c89).

Thank you.
```

**Comment by nemequ** [comment]
The NEON stuff is likely going to fail on some architectures because of aliasing.  This was fixed for Intel in #72, but not NEON.  I'm just going to disable the NEON tests for now; I'll re-enable them once NEON is fixed.

**Comment by junaruga** [comment]
Okay. Thanks for checking!
